reproduce-figures/README.md

```diff
@@ -1,10 +1,75 @@
 This directory contains a script and certain data files necessary for reproducing manuscript figures, tables, and analyses.
+It contains the following contents:
 
-The script `prepare-scpca-portal-data.R` organizes and prepares data obtained from the ScPCA Portal and S3 needed to reproduce figures and analyses.
-**CAUTION: Running this script requires at least 170 GB of free disk space.**
+* The script `prepare-scpca-portal-data.R` prepares data obtained from the ScPCA Portal and AWS S3 needed to reproduce figures, tables, and the bulk RNA-Seq analysis.
+Please see instructions below for usage.
   * This script uses functions provided in `utils.R`
-  * This script also uses the `scratch` directory during file processing.
+  * This script also uses the `./scratch` directory during file processing
+* TSV files `FigS1B-gene-expression-data.tsv` and `FigS1C_FigS1D-cell-metrics-data.tsv` are used as input to the script `../scripts/FigS1B-D_method-metrics-comparison.R` which creates Figures S1-D.
+  * Note that this script expects these TSV files to be present in this directory; please do not move them.
+
+**Caution: This process script requires at least 170 GB free storage space.**
 
 ## Instructions
 
-_Forthcoming._
+To reproduce figures and analyses in this repository, you will need to organize and prepare all data needed for input.
+Follow the steps below to do so:
+
+### Step 1: Set up the `renv` environment
+
+This repository uses [`renv`](https://rstudio.github.io/renv/) to manage R packages and dependencies.
+To set up the `renv` environment, open the the `scpca-paper-figures.Rproj` project in RStudio and run `renv::restore()` to install all packages in the provided `renv.lock` file.
+Note that this project uses R version `4.4.0`.
+
+Please refer to the [Introduction to `renv` vignette](https://rstudio.github.io/renv/articles/renv.html) for more information about getting started with `renv`.
+
+### Step 2: Download files from the ScPCA Portal
+
+You will need to download the following files from the ScPCA Portal <https://scpca.alexslemonade.org/>:
+
+1. All ScPCA projects listed in the [Project Whitelist file](../sample-info/project-whitelist.txt)
+  * On each project page (<https://scpca.alexslemonade.org/projects/SCPCPXXXXXX>), download each project with the following options selected:
```

Comment:
I think I might describe how to construct the URL to each project rather than providing a URL that doesn't work. Maybe like this: 

```suggestion
  * Use the project identifiers listed in the project whitelist file to navigate to each project page. 
For example, to navigate to `SCPCP000001`, use the URL: <https://scpca.alexslemonade.org/projects/SCPCP000001>. 
* Download each project using the `Download Project` button with the following options selected: 
```